### PR TITLE
fix: proxy settings page stuck

### LINKED ISSUE #9379: Proxy Settings stucked in "System" mode
### Bug description

I have installed tje 1.13.2 podman desktop version with 5.1.1 podman version (not the latest) and I use my machine sometimes with VPN&Proxy sometimes without
I have to switch of Proxy States. With new versino the Proxy Settings pannel has changed. As previous version of Desktop (1.12) seems to detect proxy configuration from OS I tried to switch to "System" Proxy settings but now it is impossible de switch back to "Manual" or "Disabled".
I got to settings, proxy settings, switch to manual for example, graphically the menu shows "manual" but no widget to enter proxy settings appears and if close settings and come back, "system" is still selected

### Operating system

Windows 11

### Installation Method

Installer from website/GitHub releases

### Version

1.13.2

### Steps to reproduce

- Go to settings
- Go to "Proxy"
- Set Proxy setting to "System"
- Close "Settings"
- Maybe you have to restart padman
- Go to settings
- Go to "Proxy"
- Set Proxy setting to "Manual"
- You can see that nothing change in the panel, no widget added to enter proxy settings
- Close settings pannel
- Go to settings
- Go to "Proxy"
- The settings is still "System" instead of manual

### Relevant log output

_No response_

### Additional context

Image

### PR DESCRIPTION
**Pull request by jeffmaury**
Fixes #9379

### What does this PR do?

Update the Proxies preference page

### Screenshot / video of UI

N/A

### What issues does this PR fix or reference?

#9379 

### How to test this PR?

Go to the proxies preference page and change to either manual or disabled

- [ ] Tests are covering the bug fix or the new feature

### CODE REVIEW

File: packages/renderer/src/lib/preferences/PreferencesProxiesRendering.svelte [hunk 1]
```diff
@@ -73,7 +73,7 @@ function validate(event: any) {
       </select>
     </label>
 
-    {#if proxySettings}
+    {#if proxyState !== ProxyState.PROXY_SYSTEM}
```
Comment (axel7083) [inline packages/renderer/src/lib/preferences/PreferencesProxiesRendering.svelte]: if I have selected `manual` then switch back to system, I will not see the update button, therefore I will not be able to update the settings ?

**Review by axel7083 — CHANGES_REQUESTED** [review]
image

The page crash on my machine

image

Because this is not the real type

**Review by axel7083 — CHANGES_REQUESTED** [review]
image

The page crash on my machine

image

Because this is not the real type
https://github.com/containers/podman-desktop/blob/cd668fe99f28d75c4029f5479e581db212c0ed8f/packages/renderer/src/lib/preferences/PreferencesProxiesRendering.svelte#L12C5-L12C18

it is `let proxySettings: ProxySettings | undefined;` and we are reading the value of undefined for the inputs

**Review by axel7083 — APPROVED** [review]
If we are in a hurry to release this is an acceptable solution 👍 , and we could do follow up PR to improve the design.

We have a dedicated issue for testing https://github.com/containers/podman-desktop/issues/9525

## UX remark

image

The `Update` button is always active, but should be disabled after we save: if we have nothing to update , the button should be disabled, this is missleading to always see it visible.

I still believe that disabled and system should hide the inputs and automatically save through the select on:change property. Avoiding the user to have to click on update after  setting to system or disabled.

**Comment by jeffmaury** [comment]
> If we are in a hurry to release this is an acceptable solution 👍 , and we could do follow up PR to improve the design.
> 
> We have a dedicated issue for testing #9525
> ## UX remark
> 
> image
> 
> The `Update` button is always active, but should be disabled after we save: if we have nothing to update , the button should be disabled, this is missleading to always see it visible.
> 
> I still believe that disabled and system should hide the inputs and automatically save through the select on:change property. Avoiding the user to have to click on update after setting to system or disabled.

That is to be discussed and out of scope of this PR.

**Review by gastoner — APPROVED** [review]
LGTM

**Pull request merged**